stock_virtual_available_adj/tests/test_product.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestProduct(common.TransactionCase):
+    """
+        This tests allows to check website sale available quantity for product.
+    """
```

Comment:
```suggestion
```
I think we should remove this comment as it's not carrying much meaningful message.